pom.xml

```diff
@@ -419,9 +419,22 @@
                             <failFast>true</failFast>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                     <version>[1.8.0-241,1.9)</version>
+                                    <message>Invalid JDK version, minimum required is 1.8.0_241.</message>
+                                </requireJavaVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
```

Comment:
Could you please wrap this up in a Maven profile that is only activated when the JDK is 8? Thanks! :)

Reply:
Actually, my bad, you could have multiple versions defined.